tests/unit/services/scroll-activity-test.js

```diff
@@ -146,3 +147,99 @@ test('unsubscribe', function (assert) {
     });
   });
 });
+
+test('event triggered for horizontal window scroll', function (assert) {
```

Comment:
Can you add another test that triggers both a vertical and horizontal scroll at the same time and assert that only 1 event gets fired?